Formula/j/jjui.rb

```diff
@@ -0,0 +1,23 @@
+class Jjui < Formula
+  desc "TUI for interacting with the Jujutsu version control system"
+  homepage "https://github.com/idursun/jjui"
+  url "https://github.com/idursun/jjui/archive/refs/tags/v0.8.8.tar.gz"
+  sha256 "66963f2b091855d1e4116bdf45dbbde728ff6b2a65e4adb879e32249a3693242"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  depends_on "jj"
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X main.Version=#{version}
+    ].join(" ")
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/jjui"
+  end
+
+  test do
+    assert_equal version.to_s, shell_output("#{bin}/jjui -version").strip
```

Comment:
```suggestion
    assert_match version.to_s, shell_output("#{bin}/jjui -version")
```

Reply:
This is good to make sure the binary is reporting the correct version. We should also add a second test to verify some kind of functionality of the application.

Reply:
@chenrui333 also pointed it out, I'll find something.

Reply:
Now I also test for failure in the absence of a jujutsu repository, matching part of the correct error string.
It was tricky to get the strings equal, with the current directory path being escaped.

Better would be to spin up a jj repository, launch the program, manipulate stdin to make the TUI do a non-trivial task, quit and assert the correct behaviour.
I can try if needed.